src/parse/init.go

```diff
@@ -23,6 +23,12 @@ type aspParser struct {
 
 // newAspParser creates and returns a new asp.Parser.
 func newAspParser(state *core.BuildState) *asp.Parser {
+	p := GetParserWithBuiltins(state)
+	return p
+}
+
+// GetParserWithBuiltins returns a asp.Parser object with all the builtins loaded
```

Comment:
It's idiomatic to call this `NewParser...` if it's creating a new one. `Get` implies that one already exists.